cookbooks/bcpc/recipes/cobbler.rb

```diff
@@ -37,7 +37,7 @@
 chef_vault_secret "cobbler" do
   data_bag 'os'
   raw_data({ 'web-password' => web_passwd, 'root-password' => root_passwd, 'root-password-salted' => root_passwd.crypt("$6$" + rand(36**8).to_s(36)) })
-  admins node[:hostname]
+  admins node[:fqdn]
```

Comment:
Amit and I discussed why this will not break backwards compatibility and it was pointed out older bootstrap nodes did not know their domain so their fqdn is equal to their hostname.